packages/api/src/resolvers/user_personalization/index.ts

```diff
@@ -17,19 +17,12 @@ export const setUserPersonalizationResolver = authorized<
   MutationSetUserPersonalizationArgs
 >(async (_, { input }, { authTrx, uid }) => {
   const newValues = input as Omit<SetUserPersonalizationInput, 'digestConfig'>
-  const digestValues = input.digestConfig
-    ? {
-        digestConfig: () => {
-          return JSON.stringify(input.digestConfig)
-        },
-      }
-    : {}
   const result = await authTrx(async (t) => {
     return t.getRepository(UserPersonalization).upsert(
       {
         user: { id: uid },
+        digestConfig: input.digestConfig as any,
```

Comment:
I wonder if changing the entity class attribute type could be easier

Reply:
Something like `digestConfig: { channels: string[] }`

Reply:
Yeah I have tried but fighting a bit with all the Maybe type stuff.